cli/deploy.ts

```diff
@@ -54,27 +53,30 @@ const eventWithSectionAndMethod = (events: EventRecord[], section: string, metho
 };
 
 // Given a list of schema names, attempt to create them with the chain.
-const createSchemas = async (schema_names: string[]) => {
+const createSchemas = async (schemaNames: string[]) => {
   const promises: Promise<void>[] = [];
   const api = await getFrequencyAPI();
   const signerAccountKeys = getSignerAccountKeys();
   // Mainnet genesis hash means we should propose instead of create
   const shouldPropose =
     api.genesisHash.toHex() === "0x4a587bf17a404e3572747add7aab7bbe56e805a5479c6c436f07f36fcc8d3ae1";
 
-  if (shouldPropose && schema_names.length > 1) {
+  if (shouldPropose && schemaNames.length > 1) {
     console.error("Proposing to create schemas can only occur one at a time. Please try again with only one schema.");
     process.exit(1);
   }
 
   // Retrieve the current account nonce so we can increment it when submitting transactions
-  let nonce = (await api.rpc.system.accountNextIndex(signerAccountKeys.address)).toNumber();
+  const baseNonce = (await api.rpc.system.accountNextIndex(signerAccountKeys.address)).toNumber();
+
+  for (const idx in schemaNames) {
+    const schemaName = schemaNames[idx];
+    const nonce = baseNonce + Number(idx);
```

Comment:
The base nonce and this ensure that the schemas are always deployed in the same order.